src/checkers/inference/DefaultSlotManager.java

```diff
@@ -342,6 +342,15 @@ public SourceVariableSlot createSourceVariableSlot(AnnotationLocation location,
         return sourceVarSlot;
     }
 
+    @Override
+    public VariableSlot createPolymorphicInstanceSlot(AnnotationLocation location, TypeMirror type) {
+        // TODO: For now, a polymorphic instance slot is just equivalent to a non-insertable
+        //  source variable slot. We may consider changing this implementation later.
+        SourceVariableSlot slot = createSourceVariableSlot(location, type);
+        slot.setInsertable(false);
```

Comment:
Maybe add a comment here that instead of calling `setInsertable` here we could add that to an overload of `createSourceVariableSlot`. I assume you're not doing that yet as we might change the implementation and then not need that overload?

Reply:
Yes, I thought we are going to replace it anyways. I just added a private overload of `createSourceVariableSlot` to make future deprecation of `setInsertable` easier.